src/components/PageWithSidebar/PageWithSidebar.tsx

```diff
@@ -89,7 +92,11 @@ export const PageWithSidebar = ({
 
   return (
     <Drawer isExpanded={isExpanded} isInline={windowWidth > WINDOW_WIDTH_THRESHOLD_PX}>
-      <DrawerContent panelContent={panelContent}>{children}</DrawerContent>
+      <DrawerContent panelContent={panelContent}>
+        <DrawerContentBody>
+          <div className={styles['drawer-content-body']}>{children}</div>
```

Comment:
`drawer-content-body` can be misleading, something like `drawer-content-body-in` or `drawer-content-body-inner` can be more precise